python/pip_install/extract_wheels/lib/BUILD

```diff
@@ -41,6 +41,19 @@ py_test(
     ],
 )
 
+py_test(
+    name = "bazel_test",
```

Comment:
nit: Think this test name (and associated file) could be more specific given it's only testing the new wheel filegroup. `whl_filegroup_test`?

Reply:
Yeah I get that, the idea was the name should be more general to allow expanding to include other tests of the bazel.py module. Happy to change it for the moment.